submissions/LuckyDnepr/Hooli-popup-task/index.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles-menu.css">
+    <title>Hooli menu</title>
+</head>
+
+<body>
```

Comment:
@LuckyDnepr Good job ^^

Let's make some changes.

Please, use a template similar to the one in the example:
![image](https://user-images.githubusercontent.com/50628589/183515945-b68e6e43-1eb7-43bb-b0e1-6811a4e6ee13.png)

It doesn't have to be the same, but have similar navigation elements like links and popup to have opportunity navigate using keyboard.